In [1]:
#import necessary libraries
import math
from scipy.stats import norm

In [2]:
#Given data
sample_mean = 3050
population_mean = 600
standard_deviation = 25
sample_size = 25

In [3]:
# Calculate the Z-test statistic
z_statistic = (sample_mean - population_mean) / (standard_deviation / math.sqrt(sample_size))


In [4]:
print(f"Z-test statistic: {z_statistic:.2f}")

Z-test statistic: 490.00


In [5]:
#given alpha value
alpha = 0.05
# Critical values for two-tailed test
z_critical = norm.ppf(1 - alpha / 2)  # 1 - alpha/2 for right-tail
print(f"Critical Z-value (Two-tailed): ±{z_critical:.2f}")

Critical Z-value (Two-tailed): ±1.96


z-statistic is greater than the critical z-value based on the significance level, the null hypothesis is rejected.

Conclusion :- Here z-statistic of 490 is an extremely large value, indicating that the sample mean is far from the hypothesized population mean. 
This suggests that the null hypothesis is highly unlikely to be true, and the observed sample mean is statistically significant.test statistic 
𝑧 = 490 far exceeds the critical z-value, providing extremely strong evidence to reject the null hypothesis.
   